From linux-usb-owner@vger.kernel.org Sat Mar  8 00:04:16 2008
From: "Robert P. J. Day" <rpjday@crashcourse.ca>
Date: Sat, 8 Mar 2008 03:04:05 -0500 (EST)
Subject: USB: Remove EXPERIMENTAL designation from USB MDC800 support.
To: linux-usb@vger.kernel.org
Message-ID: <alpine.LFD.1.00.0803080302310.7104@localhost.localdomain>

Since support for the USB Mustek MDC800 Digital Camera has apparently
been around since the beginning of the git repository, it's safe to
assume it's no longer experimental.

Signed-off-by: Robert P. J. Day <rpjday@crashcourse.ca>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/usb/image/Kconfig |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

--- a/drivers/usb/image/Kconfig
+++ b/drivers/usb/image/Kconfig
@@ -5,8 +5,8 @@ comment "USB Imaging devices"
 	depends on USB
 
 config USB_MDC800
-	tristate "USB Mustek MDC800 Digital Camera support (EXPERIMENTAL)"
-	depends on USB && EXPERIMENTAL
+	tristate "USB Mustek MDC800 Digital Camera support"
+	depends on USB
 	---help---
 	  Say Y here if you want to connect this type of still camera to
 	  your computer's USB port. This driver can be used with gphoto 0.4.3
